js-i2b2/cells/PM/PM_admin_ctrlr.js

```diff
@@ -27,9 +27,12 @@ i2b2.PM.view.admin.yuiTreeNodeAPPROVALS = false;
 i2b2.PM.view.admin.yuiTreeNodeCELLS = false;
 i2b2.PM.view.admin.Resize = function(e){
 //	var t = $('pmNavTreeview');
-	var ds = document.viewport.getDimensions();
-	var w = ds.width;
-	var h = ds.height-50;
+	//var ds = document.viewport.getDimensions();
```

Comment:
I don't understand where the patches to this file came from and what they have to do with CAS.